metaschema-java-binding/src/main/java/gov/nist/secauto/metaschema/binding/io/yaml/YamlFactoryFactory.java

```diff
@@ -27,18 +27,30 @@
 package gov.nist.secauto.metaschema.binding.io.yaml;
 
 import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactoryBuilder;
 import com.fasterxml.jackson.dataformat.yaml.YAMLGenerator;
 
+import org.yaml.snakeyaml.LoaderOptions;
+
 public final class YamlFactoryFactory {
+  public static final int CODEPOINT_LIMIT = 2 * 1024 * 1024 * 1024; // 2 GB
```

Comment:
Open question, not a strong recommendation: is there any substantive value in making it tuned lower and making it easily increased by a runtime variable or is that ill-advised? Either way, I will open an ADR in liboscal-java to document the decision and implementation guidance, thanks.